patches/chrome-browser-BUILD.gn.patch

```diff
@@ -1,8 +1,17 @@
 diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
-index d730660c086f9a47d524708b2b477501e257bfaa..8f7b2e5019dde9ad232e128aca9b45f0475c5203 100644
+index d730660c086f9a47d524708b2b477501e257bfaa..f24fa8e19764020f66e57cc695646d65c9ab2812 100644
 --- a/chrome/browser/BUILD.gn
 +++ b/chrome/browser/BUILD.gn
-@@ -2226,6 +2226,7 @@ jumbo_split_static_library("browser") {
+@@ -1898,6 +1898,8 @@ jumbo_split_static_library("browser") {
+     "win/ui_automation_util.h",
+   ]
+ 
++  sources += brave_notifications_sources
```

Comment:
this is not necessary, we already have a dep here for `//brave/browser:browser_process` to handle this kind of stuff

Reply:
Fixed